packages/constants/config.ts

```diff
@@ -8,6 +8,6 @@ export interface Config {
 
 export const testnetConstants: Config = {
   grpcEndpoint: 'https://grpc.testnet.penumbra.zone',
-  indexedDbVersion: 17,
+  indexedDbVersion: 18,
```

Comment:
We must increment the indexed-db version every time we change indexed-db's structure